splink/estimate_u.py

```diff
@@ -115,12 +235,18 @@ def estimate_u_values(linker: "Linker", target_rows):
 
     sql = compute_new_parameters_sql(settings_obj)
     linker._enqueue_sql(sql, "__splink__m_u_counts")
```

Comment:
Unrelated to what I was doing here, but noticed this line uses the original `linker` rather than the copied `training_linker` - not sure if this was intentional or not, so left as-is for now